reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    while state.values.t &lt; L :
</pre><p>I assume that <code>t</code> here is the internal state variable from <code>scikits</code>, but the natural variable is <code>z</code> . I think it'd be worth adding a comment here to explain that.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Added a comment.</p>